manifest/schema1/manifest_test.go

```diff
@@ -79,7 +86,7 @@ func genEnv(t *testing.T) *testEnv {
 
 	name, tag := "foo/bar", "test"
 
-	m := Manifest{
+	invalid := Manifest{
```

Comment:
Given the check added was specifically checking for a length difference between fsLayers and history, we should have a test that looks for that.